### DISCUSSION TITLE
Cursorstyle same for all themes, low contrast

### DISCUSSION THREAD

**Discussion by codingluke** [LazyVim/LazyVim#3164] [Q&A] [answered]
Hi there,

It seems that the cursorstyle including the colors (not the cursorline) are somehow always the same for all themes. For my taste the contrast is to low. Now I have the question whether it is a configuration issue on my side or does somebody else experience the same?

Here some screenshots:

Astrothem
![Screenshot_2024-05-15_000789](https://github.com/LazyVim/LazyVim/assets/643436/398dd83a-3ca0-4977-9e97-12ea2bc3ff3f)

tokyonight
![Screenshot_2024-05-15_000788](https://github.com/LazyVim/LazyVim/assets/643436/efdd3dd5-1f39-45f2-b0a0-79e0d487deb3)

kanagawa
![Screenshot_2024-05-15_000790](https://github.com/LazyVim/LazyVim/assets/643436/06f70cec-c09e-4733-8bec-306a87d80e51)

Beste and THX

**Comment by folke** [comment, marked as answer]
that's a terminal setting

**Reply by codingluke** [reply]
Thx a lot you are right. It was the kitty theme. Good to know that it has an influence to nvim. I was not aware of this.

**Reply by codingluke** [reply]
FYI: This is indeed a rabbit hole mentioned on neovim and kitty issues. I found [this solution](https://github.com/neovim/neovim/issues/12626#issuecomment-799077796) which works for the cursor background, but not for the text. I don't want to bother you to much. Just, when you would have a solution for yourself, I would be highly interested in it (as far you use kitty too).

**Reply by folke** [reply]
In kitty, I use `cursor none`. This will color the cursor based on the underlying color. It's pretty neat
